tests/components/resources/pipelines/test-distributed-defaults/defaults_development.yaml

```diff
@@ -0,0 +1,3 @@
+helm-app:
```

Comment:
is it on purpose that this has the exact same contents as `defaults.yaml`?

Reply:
The content is not important for the tests. I checked the order of the files picked up. The snapshot tests are already covering the merge functionality. Yet, I have changed it, so it should now be less confusing.